spec/till_spec.rb

```diff
@@ -0,0 +1,38 @@
+require 'till'
+
+describe Till do
+  it{is_expected.to respond_to :place_order}
```

Comment:
Space missing to the left of {.<br>Space missing inside {.<br>Space missing inside }.